.rubocop.yml

```diff
@@ -10,6 +10,7 @@ AllCops:
     - "tmp/**/*"
     - "vendor/bundle/**/*"
     - "vendor/bundle/**/.*"
+  NewCops: disable
```

Comment:
force push destroyed my comment :cry: 

As I was saying, I' d prefer to have them all listed and enabled/disabled. I think it's a nice change in rubocop to keep the configuration explicit while making it a choice to opt in. Super happy to have them all just listed as disabled and someone can upgrade them later down the line. IIrc I liked most of them though :)

Reply:
I'm sorry :crying_cat_face:... I didn't know force-push could deleted comments, I thought it should only render them out of date :open_mouth:, github bug?

Anyways, I also think it's a lovely change in rubocop. I was just lazy and my whole point was to be able to try out latest rubocop on a real project in order [to get a bug fix tested in the realworld](https://github.com/rubocop-hq/rubocop/pull/8176) :stuck_out_tongue_winking_eye:

I will add all the new cops explicitly, should I enable them or keep them disabled?

Reply:
I think github bug yeah, I hadn't submitted the review yet... no clue :shrug: 

Feel free to keep them disabled. Whatever you prefer honestly. If you keep them disabled we can keep this small and then enable them in a follow up PR.

Reply:
Also: cool bugfix :tada:

Reply:
I left the new cops disabled for now :+1:.